src/analog.jl

```diff
@@ -67,8 +64,10 @@ read_analog_cfunctions = Dict{Type,Function}(
     UInt16 => ReadBinaryU16,
     UInt32 => ReadBinaryU32 )
 
-function read(t::AITask, precision::DataType, num_samples_per_chan::Integer = -1)
-    num_channels = getproperties(t)["NumChans"][1]
+function read(t::AITask, num_samples_per_chan::Integer = -1, precision::DataType = Float64)
```

Comment:
why the change in the input arguments?  it's another breaking change.

Reply:
Change in order was due to the reasoning that someone would be more likely to have the default datatype be Float64 than for someone to have the default n_samples. Therefore you get the convenience of specifying datatype only when it's different instead of any time the datatype and/or n_samples is different. I was actually considering just dropping support for the other data types in the higher level call and just leaving the lower level calls if someone wanted to be weird :stuck_out_tongue_winking_eye:

Reply:
agree that changing the order makes sense, and having a default for precision is great too